pkg/console/subresource/authentication/cluster.go

```diff
@@ -0,0 +1,39 @@
+package authentication
+
+import (
+	"fmt"
+
+	config "github.com/openshift/api/config/v1"
+)
+
+const (
+	CLIOIDCClientComponentName = "openshift-console/cli"
+	OCLoginCommand             = "oc login"
+	ExecPluginFlag             = "--exec-plugin"
+	OCExecPlugin               = "oc-oidc"
+	ClientIDFlag               = "--client-id"
+	IssuerURLFlag              = "--issuer-url"
```

Comment:
is this API server or external issuer url?. Because `oc login` extracts issuer url from API server's well-known endpoint and there is no `--issuer-url` flag. Additionally, where is setting API server endpoint of the cluster?.

Reply:
This was a misunderstanding on my part. I need to switch this to `--server` flag and use the API server endpoint.